rakelib/tests.rake

```diff
@@ -67,31 +68,35 @@ begin
     namespace :linux do
       # Run RSpec tests that do not require WinRM
       desc ''
-      RSpec::Core::RakeTask.new(:fast) do |t|
-        t.rspec_opts = '--tag ~winrm --tag ~lxd_transport --tag ~windows_agents --tag ~puppetserver ' \
-                       '--tag ~puppetdb --tag ~omi --tag ~windows --tag ~kerberos --tag ~expensive ' \
-                       '--tag ~lxd_remote --tag ~podman'
+      RSpec::Core::RakeTask.new(:unit) do |t|
+        t.pattern = "spec/unit/*_spec.rb"
+        t.rspec_opts = '--tag ~winrm --tag ~windows'
       end
 
       # Run RSpec tests that are slow or require slow to start containers for setup
       desc ''
-      RSpec::Core::RakeTask.new(:slow) do |t|
-        t.rspec_opts = '--tag puppetserver --tag puppetdb --tag expensive'
+      RSpec::Core::RakeTask.new(:integration) do |t|
+        t.pattern = "spec/integration/*_spec.rb"
```

Comment:
```suggestion
        t.pattern = "spec/integration/**/*_spec.rb,spec/bolt_server/**/*_spec.rb,spec/bolt_spec/**/*_spec.rb"
```

We want to include tests in sub-directories (`**`) and also the bolt_server & bolt_spec tests.